src/client/components/menu/MenuAuthenticated.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { MemoryRouter } from 'react-router-dom';
+
+import { Menu } from './Menu.component';
+
+export default { title: 'Components / MenuAuthenticated' };
```

Comment:
Please put both stories under the "Menu" folder.

```suggestion
export default { title: 'Components / Menu' };
```